### DISCUSSION TITLE
Support W3C Trace Context

### DISCUSSION THREAD

**Discussion by knutwannheden** [Azure/api-management-developer-portal#2583] [Feature request]
I think it would be very useful if the developer portal would provide some option to automatically add a W3C `traceparent` header with a randomly generated value.

An accompanying `tracestate` header could also be interesting, but it would probably need to be configurable. In our case we would like to set it to some fixed value.

**Comment by ghost** [comment]
@knutwannheden, thank you for opening this issue. We will triage it within the next few business days.

**Comment by ghost** [comment]
@knutwannheden, thank you for requesting this feature.
